### PR TITLE
one more line

### PR DESCRIPTION
**Pull request by singnurkar**
Hey @Hebime I added the awesome amazing one more line feature as requested. Please tell me about how amazing my implementation is and any further praise you'd like to add. Cheers

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1,2 @@
+one more line
```
Comment (Hebime) [inline test.txt]: What is this? It's too complicated!

**Comment by Hebime** [comment]
Hey, I think we should 2 lines instead

File: test.txt (hunk 1 above)
Comment (singnurkar) [inline test.txt]: What if I change "one" to 1. I feel like this reduces complexity ten fold. All of a sudden I have absolute clarity of mind and soul. Deep breath.

**Comment by singnurkar** [comment]
4391652422_0934866e5a_n

Hey I just got a design for the new duckface interface from Axel and it looks amazing. I think we can get this coded out quickly and onto prod.

File: test.txt (hunk 1 above)
Comment (Hebime) [inline test.txt]: Perfect! My mind can now be at ease. I haven't been sleeping over this commit for months!

**Comment by Hebime** [comment]
The duckface looks great! It shouldn't take more than 4 seconds to implement this. Let's do it!

**Comment by singnurkar** [comment]
Great, I'll get it done ASAP

**Comment by Hebime** [comment]
@singnurkar that duckface has never looked better.

File: test.txt [hunk 2]
```diff
@@ -1 +1,3 @@
+one more line 
 hello this is a test
+i duckface at your test
```
Comment (Hebime) [inline test.txt]: testing add for duckface

**Comment by singnurkar** [comment]
Merged a little too prematurely there eh, boyo?

**Comment by singnurkar** [comment]
We need some street cred.

**Comment by ilmm** [comment]
Why am I getting all these notifications in double?

--
Iain Lowe | CTO | Made Media Inc. 
T: 514.507.2990 | F: 514.747.2736
www.mademediacorp.com

This e-mail communication is confidential and legally privileged. If you are not the intended recipient, please be informed that it is strictly forbidden to divulge, distribute or copy this message. If you are not the intended recipient, please notify me by telephone or by return email immediately. Thank you.

**Pull request merged**